### PR TITLE
Control for apps

### PR DESCRIPTION
**Pull request by mzia**
@afeld : I added some notes to this page. See what you think.  -mz

### CODE REVIEW

File: _pages/ato/controls.md [hunk 1]
```diff
@@ -43,6 +49,8 @@ We use AWS IAM to manage authorization and authentication.
 * [IA-2 (2) Identification and Authentication (Organizational Users) &#124; Network Access to Non-Privileged Accounts](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=IA-2#enhancement-2)
 * [IA-2 (12) Identification and Authentication &#124; Acceptance of PIV Credentials](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=IA-2#enhancement-12): consult with the Infrastructure Team for Applicability
 
+_Note: If you are using 18F's [cloud.gov](https://cloud.gov/) PaaS, the hosted applications **may** inherit the security controls listed above, unless there are external application specific IAM control dependencies._
```
Comment (afeld) [inline _pages/ato/controls.md]: Can we simplify this wording a bit?

https://pages.18f.gov/plain-language-tutorial/

File: _pages/ato/controls.md [hunk 2]
```diff
@@ -53,6 +61,8 @@ See the [security scanning](../../security/scanning/) page for more details.
 * [SI-2 Flaw Remediation](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=SI-2)
 * [SI-10 Information Input Validation](https://web.nvd.nist.gov/view/800-53/Rev4/control?controlName=SI-10)
 
+_Note: If you are using 18F's [cloud.gov](https://cloud.gov/) PaaS, the applications will **NOT** inherit the security controls listed above, thus it will be up to your enterprise team to support these controls._
```
Comment (afeld) [inline _pages/ato/controls.md]: The target audience of this site is 18F...who is our "enterprise team"?

**Comment by afeld** [comment]
Thanks for doing this! Would it be fair to say that page contains the full set of controls that would be relevant for an application hosted on AWS? Since the (strong) recommendation is to use cloud.gov, I'm wondering if we should distill the page down to say "here are the controls you are responsible for on cloud.gov – if you are hosting elsewhere and need an ATO, your life is going to be a lot harder – consult your designated Authorizing Official." Thoughts?

**Comment by mzia** [comment]
Yes, that is the direction I'd navigate the users towards: take advantage
of cloud.gov otherwise ATO will get harder.  Any controls not inherited
from cloud.gov will have to be documented by the app owner (captured in our
Control Masonry SSP) as we discussed yesterday with Jacob, which is
essentially what I was trying to get at with my notes.

So, having said that your question "Would it be fair to say that page
contains the full set of controls that would be relevant for an application
hosted on AWS?"

I'd state that these are guidelines for 18F/GSA deployments and IFF your
app can take advantage of all of the FedRAMP AWS controls
<https://www.fedramp.gov/files/2015/03/FedRAMP-Control-Quick-Guide-Rev4-FINAL-01052015.pdf>
at
the targeted assurance level (Low-Moderate). Using AWS service <> FedRAMP
AWS <https://aws.amazon.com/compliance/fedramp/>services.

On Tue, Feb 9, 2016 at 12:15 PM, Aidan Feldman <notifications@github.com>
wrote:

> Thanks for doing this! Would it be fair to say that page contains the full
> set of controls that would be relevant for an application hosted on AWS?
> Since the (strong) recommendation is to use cloud.gov, I'm wondering if
> we should distill the page down to say "here are the controls you are
> responsible for on cloud.gov – if you are hosting elsewhere and need an
> ATO, your life is going to be a lot harder – consult your designated
> Authorizing Official." Thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/before-you-ship/pull/124#issuecomment-181962902>.
>



-- 
Mossadeq Zia, CISSP
GSA-18F, Innovation Specialist | @18F
1800 F St NW, Washington, DC 20405
mobile: 202-256-2063
mail: mossadeq.zia@gsa.gov

File: _pages/ato/controls.md [hunk 3]
```diff
@@ -53,6 +61,8 @@ See the [security scanning](../../security/scanning/) page for more details.
...
+
 ### Monitoring
 
 One or more of CloudWatch, CloudTrail, New Relic, Splunk, etc.
```
Comment (DavidEBest) [inline _pages/ato/controls.md]: Is New Relic cloud.gov-wide, or does each application need it for monitoring?

Comment (afeld) [inline _pages/ato/controls.md]: It needs to be set up by the project teams, yes. https://github.com/18F/before-you-ship/issues/125

**Pull request merged**